lib/components/Editor/index.jsx

```diff
@@ -141,6 +141,14 @@ const Editor = (
     nextContentUpdater(nextContent);
   }, [markdownMode]);
 
+  useEffect(() => {
+    if (initialValue === "") {
+      console.warn(
+        "Empty value (initialValue) in needtoEditor should be '<p></p>' instead of an empty string."
+      );
+    }
+  }, [initialValue]);
+
```

Comment:
- It is always better to suppress the warnings in production.
- It is better to catch `null` and `undefined` in addition to empty string as well. The `isNilOrEmpty` method is defined in `lib/utils.common.js` file.

@mohit-s96 _a please make the mentioned change.

```suggestion
    useEffect(() => {
    const isProduction = [process.env.RAILS_ENV, process.env.NODE_ENV].includes(
      "production"
    );
    if (isProduction && isNilOrEmpty(initialValue)) {
      // eslint-disable-next-line no-console
      console.warn(
        `[neeto-editor]: Empty value of "initialValue" in needtoEditor is expected to be "<p></p>" instead of "${initialValue}".`
      );
    }
  }, [initialValue]);
```